profiles/coreos/base/use.mask

```diff
@@ -1,2 +1,6 @@
 # Never enable experimental code
 kdbus
+
+# We default to python 3.6 for now
+python_targets_python3_7
+python_single_target_python3_7
```

Comment:
Note for me.
When we introduce the python 3.7 target in the future, we need to unmask 3.7 here again.